### ISSUE TITLE
Make systemd service user-specific instead of system-wide

### ISSUE THREAD

**Issue by rGunti** [geerlingguy/pi-kiosk#5]
Related to the following comment on the [YouTube video](https://www.youtube.com/watch?v=gpyYCTgJO88) (I am not the author of the comment, but I share the same view and have some information about this)

> @zvpunry1971
> 2 hours ago
> [5:45](https://www.youtube.com/watch?v=gpyYCTgJO88&t=345s) Don't throw own service files to  /lib/systemd/system/, there they belong to /etc/systemd/system/. But that is only for services that don't belong to some specific user. On a raspberry pi you have the default user that is automatically logged in. This user has its own systemd running, it is controlled with "systemctl --user ..." and has its service files in ~/.config/systemd/user/ 
> 
> That said, depending on your desktop environment (I assume raspberry pi defaults), there is another place to automatically start the script: ~/.config/autostart/ where you can put *.desktop files.

The `.service` file has to be located under `~/.config/systemd/user/` instead and `systemctl` commands have to be used with the `--user` flag, apart from that the handling should be the same.

**Comment by geerlingguy** [comment]
Agreed, since this is a userspace thing, it shouldn't apply to the global system. Permissions might even be easier (e.g. not having to do #4 and use `sudo` to copy that file).

**Comment by wulph111** [comment]
> The `.service` file has to be located under `~/.config/systemd/user/` instead and `systemctl` commands have to be used with the `--user` flag, apart from that the handling should be the same.

I tried to implement this and got the following error:

`pi@pipad:~/bin $ systemctl --user enable kiosk`

...

`Unit /home/pi/.config/systemd/user/kiosk.service is added as a dependency to a non-existent unit graphical.target.`

**Comment by geerlingguy** [comment]
Ah, in that case we could probably remove that target from `kiosk.service`, since the user account would only be logged in after the graphical target's been reached!

**Comment by sprior** [comment]
I was just working on this a little.  Starting from the kiosk.service in the repo I had to comment out the User and Group lines.  I also changed the graphical.target to default target.  I mostly got it working except that the cursor didn't hide - haven't figured out why.
